### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by utrennik**
Capture
In scope:
* create welcome page component
* add app SVG logo
* add start game button
* add connect to lobby controlled input and button
* add error messages
* start new game button triggers to open the empty modal window

Guys, i would prefer to use default styles for material-UI components, not wasting time to customize it according to our figma design (it's not so fast). Maybe we'll do it later if we'll have enough time.

### CODE REVIEW

File: client/src/components/pages/welcome-page/welcome-page.sass [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  span
+    font-weight: 700
+    color: $cadet-blue
+
+.start-game-container
+  display: flex
+  align-items: center
+  justify-content: space-between
+  margin-bottom: 50px
+  width: 330px
```
Comment (HelenBassa) [inline client/src/components/pages/welcome-page/welcome-page.sass:59]: I suggest adding the 'margin-bottom: 50px' property to the .connect-container class to correctly display the error message (by analogy with the .start-game-container class).
At a screen resolution of 1366 × 768, it is displayed like this:

File: client/src/components/pages/welcome-page/welcome-page.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { Button } from '@material-ui/core';
+import InputButton from '@components/ui/input-button/input-button';
+import pokerLogo from '@assets/images/poker.svg';
+import './welcome-page.sass';
+
+const WelcomePage = () => (
+  <div className="_container">
```
Comment (HelenBassa) [inline client/src/components/pages/welcome-page/welcome-page.tsx]: I suggest (if necessary) to replace the ***._container*** class with ***.container*** in the WelcomePage component (***._container*** was changed by me to ***.app -container***).

**Review by HelenBassa — COMMENTED** [review]
1. I suggest adding the 'margin-bottom: 50px' property to the .connect-container class to correctly display the error message (by analogy with the .start-game-container class).
2. I suggest (if necessary) to replace the ***._container*** class with ***.container*** in the WelcomePage component (***._container*** was changed by me to ***.app -container***).

File: client/src/components/pages/welcome-page/welcome-page.sass (hunk 1 above)
Comment (utrennik) [reply client/src/components/pages/welcome-page/welcome-page.sass:59]: fixed, thanks!

File: client/src/components/pages/welcome-page/welcome-page.tsx (hunk 2 above)
Comment (utrennik) [reply client/src/components/pages/welcome-page/welcome-page.tsx]: fixed, thanks!

**Review by HelenBassa — APPROVED** [review]

**Review by RealOneReal — APPROVED** [review]

File: client/src/components/pages/welcome-page/welcome-page.sass [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  font-size: 48px
+  font-weight: 700
+  color: $cadet-blue
+  margin-bottom: 50px
+  @media (max-width: 700px)
+    font-size: 40px
+
+.action-title
+  font-size: 22px
+  font-size: 23px
```
Comment (aksiantsiuk) [inline client/src/components/pages/welcome-page/welcome-page.sass]: duplication :)

File: client/src/components/ui/input-button/input-button.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import React, { ChangeEvent, FormEvent, useState } from 'react';
```
Comment (aksiantsiuk) [inline client/src/components/ui/input-button/input-button.tsx]: you don't need import React itself

File: client/src/models/types.ts [hunk 5]
```diff
@@ -5,3 +5,8 @@ export interface IWebLink {
   style: string;
   children: string | ReactNode;
 }
+
+export interface IInputButtonProps {
+  buttonText: string;
+  valueHandler: Function;
+}
```
Comment (aksiantsiuk) [inline client/src/models/types.ts]: Here best practice is describe each prop

```js
export interface IInputButtonProps {
   /*
   * The text of the button
  */
  buttonText: string;
   /*
   * The action that will be called on submit event
  */
  valueHandler: Function;
}
```

Aslo `Function` type is not clear. Better to describe it like `(string) => void`

File: client/src/styles/config.sass [hunk 6]
```diff
@@ -27,10 +27,13 @@ input, textarea
 html
 	height: 100%
 
+html,
 body
 	height: 100%
```
Comment (aksiantsiuk) [inline client/src/styles/config.sass]: Style duplication

Comment (utrennik) [reply client/src/styles/config.sass]: fixed, thanks!

File: client/src/models/types.ts (hunk 5 above)
Comment (utrennik) [reply client/src/models/types.ts]: fixed, thanks!

File: client/src/components/ui/input-button/input-button.tsx (hunk 4 above)
Comment (utrennik) [reply client/src/components/ui/input-button/input-button.tsx]: fixed, thanks!

File: client/src/components/pages/welcome-page/welcome-page.sass (hunk 3 above)
Comment (utrennik) [reply client/src/components/pages/welcome-page/welcome-page.sass]: fixed, thanks!

File: client/src/models/types.ts [hunk 7]
```diff
@@ -5,3 +5,14 @@ export interface IWebLink {
...
+
+export interface IInputButtonProps {
+  /*
+   * the text of the button
+   */
+  buttonText: string;
+  /*
+   * The action will be called on submit event
+   */
+  valueHandler: (arg0: string) => void;
```
Comment (aksiantsiuk) [inline client/src/models/types.ts]: `arg0` is it mistake in naming?

Comment (utrennik) [reply client/src/models/types.ts]: fixed to 'value'

**Pull request merged**